pkg/rancher-desktop/assets/specs/command-api.yaml

```diff
@@ -238,50 +238,63 @@ components:
           properties:
             adminAccess:
               type: boolean
-              x-rd-platforms: ['darwin', 'linux'] # Only in the specified platforms
+              x-rd-usage: Enable privileged operations
+              x-rd-platforms: [darwin, linux] # Only in the specified platforms
             debug:
               type: boolean
+              x-rd-usage: Generate more verbose logging.
             pathManagementStrategy:
               type: string
-              enum: ['manual', 'rcfiles']
-              x-rd-platforms: ['darwin', 'linux']
+              enum: [manual, rcfiles]
+              x-rd-platforms: [darwin, linux]
+              x-rd-usage: Update PATH to include ~/.rd/bin.
             telemetry:
               type: object
               properties:
                 enabled:
                   type: boolean
+                  x-rd-usage: Allow collection of anonymous statistics.
             updater:
               type: object
               properties:
                 enabled:
                   type: boolean
+                  x-rd-usage: Automatically update to the latest release.
             autoStart:
               type: boolean
+              x-rd-usage: Start app when logging in.
             startInBackground:
               type: boolean
+              x-rd-usage: Start app without window.
             hideNotificationIcon:
               type: boolean
+              x-rd-usage: Don't show notification icon.
             window:
               type: object
               properties:
                 quitOnClose:
                   type: boolean
+                  x-rd-usage: Terminate app when the main window is closed.
         containerEngine:
           type: object
           properties:
             name:
               type: string
-              enum: ['containerd', 'docker', 'moby']
-              x-rd-usage: Set engine to containerd or moby (aka docker).
-              x-rd-aliases:
-                - "container-engine"
+              # TODO "docker" setting should be a hidden alias of "moby".minimum:
```

Comment:
Chris thanked me for getting `docker` in there. We currently have aliases for option names but not values

Reply:
#4113 includes a request to make hidden aliases available.

I'm fine with silently supporting `docker` as an alias for `moby` (or vice versa), but having redundant aliases in descriptions, help texts, error messages is just useless clutter.